realm-annotations-processor/src/main/java/io/realm/processor/RealmProcessor.java

```diff
@@ -159,7 +131,8 @@ private boolean processModules(RoundEnvironment roundEnv) {
         if (moduleMetaData.shouldCreateDefaultModule()) {
             if (!createDefaultModule()) {
                 return false;
-            };
+            }
+            ;
```

Comment:
That semicolon should not be there in the first place. Good thing we caught it now :D